test/suite/test_chunkcache04.py

```diff
@@ -89,16 +90,17 @@ def test_chunkcache04(self):
             self.insert(uris[i], dataset)
 
         # As we have not flushed yet, assert we have no newly inserted chunks.
-        self.assertEqual(self.get_stat(wiredtiger.stat.conn.chunkcache_chunks_loaded_from_flushed_tables), 0)
+        stat_assert_equal(self.session, wiredtiger.stat.conn.chunkcache_chunks_loaded_from_flushed_tables, 0)
 
         # Flush the unpinned tables into the chunk cache.
         self.session.checkpoint()
         self.session.checkpoint('flush_tier=(enabled)')
 
         # Assert that chunks are not pinned.
-        self.assertEqual(self.get_stat(wiredtiger.stat.conn.chunkcache_chunks_pinned), 0)
+        stat_assert_equal(self.session, wiredtiger.stat.conn.chunkcache_chunks_pinned, 0)
 
         # Assert the new chunks are ingested.
+        time.sleep(0.5)
```

Comment:
I don't see why we wouldn't just modify get_stat() in this test to have `time.sleep(0.5)` at the start so every time we get a new stat we can wait a bit and can remove all the other 'time.sleep(0.5)'

Reply:
Good call, fixed in 963fbe0. The idea was that only the first sleep was necessary if there was a sequence of `get_stat` calls. But every `assert_stat_greater` had a sleep anyway, and I don't think it makes much difference.